Removed repeated installation instructions (#106)
ignitionrobotics/ign-gui: tutorials/01_install.md

Next Tutorial: \ref commandline

[Install](#install)

* [Binary Install](#binary-install)

* [Source Install](#source-install)

    * [Prerequisites](#prerequisites)

    * [Building from Source](#building-from-source)

# Install

We recommend following the [Binary Install](#binary-install) instructions to get up and running as quickly and painlessly as possible.

The [Source Install](#source-install) instructions should be used if you need the very latest software improvements, if you need to modify the code, or if you plan to make a contribution.

## Binary Install

> On Ubuntu Bionic, it's possible to install Ignition GUI as follows:

Reviewer: ```suggestion
On Ubuntu it's possible to install Ignition GUI as follows:
```

No need to specify an Ubuntu version, because it varies

Add OSRF packages:

    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list
    sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
    sudo apt update

Install Ignition GUI:

    sudo apt install libignition-gui<#>-dev

Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
which version you need.

## Source Install

There are currently two versions under active development:

* **Version 1**: It makes use of `QtQuick`, which allows developing more
  modern-looking apps with animations and a mobile/web feel. Requires Ubuntu

>   Bionic or higher.

Reviewer: We can remove this entire section (lines 40 to 44), because there's no distinction between the versions

### Prerequisites

#### Ubuntu Bionic 18.04 or above

Install dependencies:

    sudo apt update
    sudo apt -y install wget lsb-release gnupg
    sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
    wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
    sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
    sudo apt-get build-dep -y ignition-gui4
    sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8

Clone source code:

>     # This checks out the `default` branch. You can append `-b ign-gui#` (replace # with a number) to checkout a specific version

Reviewer: ```suggestion
    # This checks out the `master` branch. You can append `-b ign-gui#` (replace # with a number) to checkout a specific version
```

Next Tutorial: \ref commandline

[Install](#install)

* [Binary Install](#binary-install)

* [Source Install](#source-install)

    * [Prerequisites](#prerequisites)

    * [Building from Source](#building-from-source)

# Install

We recommend following the [Binary Install](#binary-install) instructions to get up and running as quickly and painlessly as possible.

The [Source Install](#source-install) instructions should be used if you need the very latest software improvements, if you need to modify the code, or if you plan to make a contribution.

## Binary Install

> On Ubuntu Bionic, it's possible to install Ignition GUI's version 1 as follows:

Reviewer: Let's try not to mention specific versions so the tutorial will need less maintenance.

```suggestion
On Ubuntu Bionic, it's possible to install Ignition GUI as follows:
```

Add OSRF packages:

    echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list

>     echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-prerelease `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-prerelease.list

Reviewer: Let's remove `prerelease`, we should have a separate tutorial for this.

    sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
    sudo apt update

Install Ignition GUI:

>     sudo apt install libignition-gui4-dev

Reviewer: Let's make this about a generic `ign-gui` version `N`. This one hasn't been released yet.

## Source Install

There are currently two versions under active development:

* **Version 0**: It makes use of `QWidgets`, which are common on traditional

>   desktop-oriented interfaces. Requires Ubuntu Xenial or Higher.

Reviewer: Let's remove the `Version 0` instructions, we're not actively supporting it anymore, and users can always check the `ign-gui0` branch.
